crypto/mbedtls/Kconfig

```diff
@@ -17,7 +17,7 @@ config MBEDTLS_VERSION
 
 config MBEDTLS_DEBUG_C
 	bool "This module provides debugging functions."
-	default n
+	default y
```

Comment:
I think the debugging functions should be enabled by default because it could increase the final firmware size if the user didn't see it is enabled. Maybe you could use "default y if DEBUG_FEATURES"